fastlane/Fastfile

```diff
@@ -93,6 +93,7 @@ def distribute_builds(
   # If we are distributing a build without a tag, i.e. a development build, we also want to update the latest build reference for distribution.
   update_latest = release_tag.nil?
   suffix = release_tag.nil? ? commit_hash : release_tag
+  normalized_version = suffix.gsub(/^v|-.*/, '')
```

Comment:
Added `v` as without that build script tried to access `Studio v1.3.0.0.appx` and not `Studio 1.3.0.0.appx`.